tooling/templatize/pkg/pipeline/run.go

```
@@ -194,3 +236,17 @@ func (rg *ResourceGroup) Validate() error {
 	}
 	return nil
 }
+
+func addInputVars(configuredInputs []Input, inputs map[string]output) map[string]any {
+	envVars := make(map[string]any)
+	for _, i := range configuredInputs {
+		if v, found := inputs[i.Step]; found {
```

should we raise an error when the step is not found?